### DISCUSSION TITLE
Use transformer model in spacy.load()

### DISCUSSION THREAD

**Discussion by venti07** [explosion/spaCy#11536] [Help: Coding & Implementations] [answered]
OSError: [E053] Could not read meta.json

I have trained and saved a BERT model with the trainer.
https://huggingface.co/docs/transformers/main_classes/trainer

How can I use this model in spacy.load()?

**Comment by thomashacker** [comment, marked as answer]
Hello,
here's a [guide](https://github.com/explosion/spaCy/discussions/10768) for loading HuggingFace models in spaCy.
